app/src/main/java/org/mozilla/tv/firefox/ext/String.kt

```diff
@@ -85,3 +85,6 @@ val String.isUriYouTubeTV: Boolean
 val String.isUriYouTubeTvVideo: Boolean
     get() = this.isUriYouTubeTV &&
             this.contains("/watch/")
+
+val String.isUriMozSignIn: Boolean
```

Comment:
nit: `isUriFxaSignIn` - both to be consistent with terminology in other files & just in case Mozilla introduces another layer of sign in (uh oh!).

Reply:
Good thinking.